kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -343,6 +344,13 @@ export class BrowserInterface {
   }
 
   async RequestGIFProcessor(data: { imageSource: string; id: string; isWebGL1: boolean }) {
+    const isChrome = !!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime)
```

Comment:
It shouldn't be better to use this check? So the gif still works on non-chrome browsers with offscreen canvas down the road? (and if firefox add support it'd fix itself)

```
if (typeof OffscreenCanvas !== "undefined") {     
    isSupported = true; 
} else {     
    isSupported = false; 
}
```

Reply:
good idea, thx!